test/Conversion/TppToXsmm/tpp-to-xsmm-identity.mlir

```diff
@@ -104,8 +107,42 @@ func.func @identity_to_xsmm(%arg0: f32, %arg1: memref<5x6xf32>) {
 // CHECK-LABEL: @identity_to_xsmm(
 // CHECK-SAME:  %[[ARG0:.+]]: memref<1x1xf32>, %[[ARG1:.+]]: memref<5x6xf32>)
 func.func @identity_to_xsmm(%arg0: memref<1x1xf32>, %arg1: memref<5x6xf32>) {
+  // m = 5
+  // n = 6
+  // ldi = 1
+  // ldo = 6
   // CHECK: %[[DISPATCH:.+]] = xsmm.unary.dispatch identity [5, 6, 1, 6](broadcast scalar dataType f32)
   // CHECK-NEXT: xsmm.unary identity(dataType f32, %[[DISPATCH]], %[[ARG0]], %[[ARG1]]) 
   tpp.identity ins(%arg0: memref<1x1xf32>) out(%arg1: memref<5x6xf32>)
   return
 }
+
+// -----
+
+// CHECK-LABEL: func.func @identity_1d
```

Comment:
There are not enough tests here. You need to cover every case in the code.
* 1D -> { 1D, 2D, 3D }, row / col bcast
* 2D -> { 2D }
* Rank all ones 1D and 2D

Also, please comment on the test (or rename the function) to say what case it is covering.